root/components/EntityTabs.js

```diff
@@ -111,9 +111,10 @@ function buildLinks(
 
   if (entityProperties.cover_art) {
     links.push(buildLink(
-      entity.cover_art_presence === 'darkened' ? l('Cover Art') : (
-        texp.l(
-          'Cover Art ({num})',
+      entity.cover_art_presence === 'darkened' ? lp('Cover art', 'plural') : (
+        texp.lp(
+          'Cover art ({num})',
+          'plural',
```

Comment:
Would `ln` be more appropriate here?

Reply:
I dunno. I think we'd still want to show "Images (1)" on the tab rather than "Image (1)" if we were using "Image/s" instead - the idea is that it's the tab for the images, but there's just one at the moment. As such, I'd expect it to be translated in plural always.